lib/server/userStore.js

```diff
@@ -0,0 +1,56 @@
+
+//A simple in-memory store of objects describing users.
+var mockUsers = [{
+  id: 'user1id',
+  name: "User 1",
+  address: "Some Lane, Some Street, Some Country"
+}];
+
+
+/**
+ * Setting up the user store to handle topics related to users.
+ *
+ * Whenever one of the topics below are published elsewhere (in this example, it is published by the http request handler in the router.js file),
+ * the mediator will execute the relevant topic below to perform some logic to satify the topic.
+ *
+ * @param mediator
+ */
+module.exports = function setUpUserStore(mediator) {
+
+  //Subscribing to the user list topic. This topic can be published by any other module/app that uses the same mediator.
+  //In this case, the `wfm:user:list` topic is published when the cloud app relieves a http request. See router.js .
+  mediator.subscribe('wfm:user:list', function() {
+
+    //Business logic to satisfy the `wfm:user:list` topic.
+    //The setTimeout is to highlight that this functionality can be asynchronous (e.g. a http request to another cloud app to get the users)
+    setTimeout(function() {
+
+      //Having the list of users, the `done:wfm:user:list` topic can be published to the mediator along with the list of users available.
+      //The `done` prefix is always used by convention to identify an topic as complete.
+      //This also shows that data can be passed to mediator topics for processing.
+      mediator.publish('done:wfm:user:list', mockUsers);
+    }, 1000);
```

Comment:
I think this timeout could be smaller to make this snappier in a possible live demo, maybe 200-300ms.